src/extension/src/coreTSModule.ts

```diff
@@ -3,31 +3,49 @@ import { ExtensionCommand } from "./constants";
 import { CoreTemplateStudio } from "./coreTemplateStudio";
 
 export class CoreTSModule extends WizardServant {
-    clientCommandMap: Map<
-        ExtensionCommand,
-        (message: any) => Promise<IPayloadResponse>
-    >;
+  clientCommandMap: Map<
+    ExtensionCommand,
+    (message: any) => Promise<IPayloadResponse>
+  >;
 
-    constructor() {
-        super();
-        this.clientCommandMap = this.defineCommandMap();
-    }
+  constructor() {
+    super();
+    this.clientCommandMap = this.defineCommandMap();
+  }
 
-    private defineCommandMap(): Map<
-        ExtensionCommand,
-        (message: any) => Promise<IPayloadResponse>
-    > {
-        return new Map([[ExtensionCommand.GetFrameworks, this.getFrameworks]]);
-    }
+  private defineCommandMap(): Map<
+    ExtensionCommand,
+    (message: any) => Promise<IPayloadResponse>
+  > {
+    return new Map([
+      [ExtensionCommand.GetFrameworks, this.getFrameworks],
+      [ExtensionCommand.GetPages, this.getPages]
+    ]);
+  }
 
-    async getFrameworks(message: any): Promise<IPayloadResponse> {
-        let result = await CoreTemplateStudio.GetExistingInstance().getFrameworks(message.payload.projectType);
-        return {
-            payload: {
-                frameworks: result,
-                isPreview: message.payload.isPreview,
-                projectType: message.payload.projectType
-            }
-        };
-    }
+  async getFrameworks(message: any): Promise<IPayloadResponse> {
+    let result = await CoreTemplateStudio.GetExistingInstance().getFrameworks(
```

Comment:
I know you didn't write this, but can this be changed to const?

Reply:
Definitely!